circuits/kimchi/src/wires.rs

```diff
@@ -19,12 +19,31 @@ pub const WIRES: [usize; COLUMNS] = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 1
 /// or a final output of the circuit, the cell references itself.
 #[derive(PartialEq, Eq, Clone, Copy, Debug, Serialize, Deserialize)]
 #[cfg_attr(test, derive(proptest_derive::Arbitrary))]
+#[cfg_attr(
+    feature = "wasm_types",
+    wasm_bindgen::prelude::wasm_bindgen
+)]
 pub struct Wire {
     // TODO(mimoo): shouldn't we use u32 since we serialize them as u32?
     pub row: usize,
     pub col: usize,
 }
 
+#[cfg(feature = "wasm_types")]
+#[cfg_attr(
+    feature = "wasm_types",
+    wasm_bindgen::prelude::wasm_bindgen
+)]
+impl Wire {
+    #[cfg_attr(
+        feature = "wasm_types",
+        wasm_bindgen::prelude::wasm_bindgen
+    )]
+    pub fn create(row: i32, col: i32) -> Self {
+        Self { row: row as usize, col: col as usize }
+    }
+}
```

Comment:
can't this function exist on the javascript side? It's pretty simple

Reply:
Javascript needs helpers like this to create rust objects in WASM, otherwise there's no way to construct a wire that could be passed back into rust calls.